node/full_node_integration_test.go

```diff
@@ -264,33 +269,41 @@ func testSingleAggregatorTwoFullNode(t *testing.T, useBlockExchange bool) {
 	node3 := nodes[2]
 
 	require.NoError(node1.Start())
+	defer func() {
+		assert.NoError(node1.Stop())
+	}()
```

Comment:
nit: can't you one line this?

Reply:
I tried this, but we can't inline this. Inlining makes the call to `Stop` immediately.